ssl/ssl_lib.c

```diff
@@ -635,7 +635,7 @@ SSL *SSL_new(SSL_CTX *ctx)
     if (RAND_get_rand_method() == RAND_OpenSSL()) {
         s->drbg =
             RAND_DRBG_new(RAND_DRBG_NID, RAND_DRBG_FLAG_CTR_USE_DF,
-                          RAND_DRBG_get0_global());
+                          RAND_DRBG_get0_public());
```

Comment:
Should we enable the auto reseeding for the ssl specific DRBG?

Reply:
It was @paulidale's whish not to do it because he feared that a reseed of the master DRBG would cause a chain reaction. See https://github.com/openssl/openssl/pull/4402#discussion_r140406758 and my reply https://github.com/openssl/openssl/pull/4402#discussion_r140435339.